Chapter 02 (#22)
tidy-survey-r/tidy-survey-book: 02-introducing-survey-data.Rmd

# Overview of Surveys {#c02}

> Creating and fielding surveys to provide estimates of a population are much more complex than simply adding a few questions to an online platform. Survey researchers can spend months, or even years, developing the study design, questions, and other methods for a single survey to ensure high quality data is collected. While this book focuses on the analysis methods of surveys, understanding the entire survey process can provide a better insight into what types of analyses should be conducted on the data. To gain a deeper understanding of survey design and implementation, there are many pieces of existing literature that we recommend reviewing in detail (e.g., {cite: Dillman book, Groves book, education book?, Tourangeau book, Asking Questions book?, others?}).

Reviewer: Other reference: Valliant, Dever, and Krauter book (for design)

The survey life cycle starts with a research topic or question of interest (e.g., what impact does childhood trauma have on health outcomes later in life). Researchers typically review existing data sources to determine if data are already available that can answer this question, as this can result in reduced burden on respondents, cheaper research costs, and faster research outcomes. However, if existing data cannot answer the nuances of the research question, a survey can be used to capture the exact data that the researcher needs.

When starting a survey, there are multiple things to consider including the study population, how the sample is selected, and the way the question is worded. However, each step and decision can impact how the results can be interpreted, specifically related to the types of error that can be introduced into the data. This chapter provides a brief introduction to the types of error to consider when conducting surveys and how these error sources can inform the interpretations of the data.

## Error Sources {#overview-error}

Generally, survey researchers consider there to be 7 main sources of error ({Cite Groves book}):

> -  Representation
>   -  **Coverage Error**: A mismatch between the target population and the sample frame available
>   -  **Sampling Error**: Error produced when selecting a sample from the sample frame (none if conducting a census)
>   -  **Nonresponse Error**: Differences between those who responded and did not respond to the survey (unit nonresponse) or to a given question (item nonresponse)
>   -  **Adjustment Error**: Error introduced during post-survey statistical adjustments
> -  Measurement
>   -  **Validity**: A mismatch between the topic of interest and the question used to collect that information
>   -  **Measurement Error**: A mismatch between what the researcher asked and how the respondent answered
>   -  **Processing Error**: Edits by the researcher to responses provided by the respondent (e.g., adjusts data based on illogical responses)

Reviewer: Need to two tabs to get the second level of the list to look how I think we want it.
```suggestion
-  Representation
    -  **Coverage Error**: A mismatch between the target population and the sample frame available
    -  **Sampling Error**: Error produced when selecting a sample from the sample frame (none if conducting a census)
    -  **Nonresponse Error**: Differences between those who responded and did not respond to the survey (unit nonresponse) or to a given question (item nonresponse)
    -  **Adjustment Error**: Error introduced during post-survey statistical adjustments
-  Measurement
    -  **Validity**: A mismatch between the topic of interest and the question used to collect that information
    -  **Measurement Error**: A mismatch between what the researcher asked and how the respondent answered
    -  **Processing Error**: Edits by the researcher to responses provided by the respondent (e.g., adjusts data based on illogical responses)
```

Almost every survey conducted will have some of each of these types of error, and researchers attempt to conduct a survey that reduces this *total survey error*. One of the most common ways to reduce total survey error is through weighting. However, weighting may result in increased error on its own (*adjustment error*). Therefore when designing surveys, researchers should carefully think about ways to reduce each of these error sources. Additionally when analyzing survey data, understanding decisions that researchers took to minimize these error sources can impact how results are interpreted. The remainder of this chapter dives into considerations for survey development and places where each of these sources of error can be considered.

## Study Design {#overview-design}

> Study design is often used to encompass multiple parts of the survey planning process including decisions on population, survey mode, and timeline. Multiple decisions made prior to the launch of the survey can assist researchers in reducing different error sources, and are fundamental in understanding how to interpret the results of the data. Knowing who and how to survey individuals depends on both the goals of the study and the feasibility of implementation.

Reviewer: ```suggestion
Study design is often used to encompass multiple parts of the survey planning process including decisions on the target population, survey mode, and timeline. Multiple decisions made prior to the launch of the survey can assist researchers in reducing different error sources, and are fundamental in understanding how to interpret the results of the data. Knowing who and how to survey individuals depends on both the goals of the study and the feasibility of implementation.
```

### Sampling Design {#overview-design-sampdesign}

> Who we want to survey is known as the *population* of interest in the study. This could be broad, such as, "all adults age 18+ living in the U.S.", or it could be a very specific population of interest based on a specific characteristic or location. For example, we may want to know about "adults age 18-24 who live in North Carolina" or "eligible voters living in Illinois". However, to survey individuals in these populations, a *sampling frame* is needed with contact information. If researchers are looking at eligible voters, the sampling frame would be the voting registry for a given state or area. For more broad populations like all adults living in the U.S., the sampling frame will most likely be imperfect. In these cases, researchers may choose to use a sample frame of mailing addresses and send the survey to households, or they may choose to use random digit dialing (RDD) and call random phone numbers. These imperfect sampling frames can result in *coverage error* where there is a mismatch between the population of interest and the list of individuals researchers can select. For example, if a researcher is looking to obtain estimates for "all adults age 18+ living in the U.S.", using a sample frame from mailing addresses will be missing specific types of individuals such as the homeless or transient populations. Additionally, many households have more than one adult living there, so researchers would need to consider how to get a specific individual to fill out the survey (called within household selection), or adjust the population to report on "U.S. households" instead of "individuals".

Reviewer: ```suggestion
Who we want to survey is known as the *population of interest*^[This is also referred to as the target population in some literature] in the study. This could be broad, such as, "all adults age 18+ living in the U.S.", or it could be a very specific population of interest based on a specific characteristic or location. For example, we may want to know about "adults age 18-24 who live in North Carolina" or "eligible voters living in Illinois". However, to survey individuals in these populations, a *sampling frame* is needed with contact information. If researchers are looking at eligible voters, the sampling frame could be the voting registry for a given state or area. For more broad populations like all adults living in the U.S., the sampling frame will most likely be imperfect. In these cases, researchers may choose to use a sample frame of mailing addresses and send the survey to households, or they may choose to use random digit dialing (RDD) and call random phone numbers. These imperfect sampling frames can result in *coverage error* where there is a mismatch between the population of interest and the list of individuals researchers can select. For example, if a researcher is looking to obtain estimates for "all adults age 18+ living in the U.S.", using a sample frame from mailing addresses will be missing specific types of individuals such as the homeless or transient populations. Additionally, many households have more than one adult living there, so researchers would need to consider how to get a specific individual to fill out the survey (called within household selection), or adjust the population to report on "U.S. households" instead of "individuals".
```

> Once the sample frame is selected, the next step is figuring out who to survey. In rare cases researchers may which to conduct a census and survey everyone on the sampling frame. However, the ability to implement a questionnaire at that scale is something few can do. Instead, researchers choose to *sample* individuals and use weights to estimate numbers in the population. There are a variety of different sampling methods that can be used, and more information on these can be found in Chapter \@ref(c05). This decision of which sampling method to use, impacts *sampling_error* and can be adjusted for in weighting.

Reviewer: ```suggestion
Once the sample frame is selected, the next step is figuring out who to survey. In rare cases researchers may which to conduct a census and survey everyone on the sampling frame. However, the ability to implement a questionnaire at that scale is something few can do. Instead, researchers choose to *sample* individuals and use weights to estimate numbers in the population. There are a variety of different sampling methods that can be used, and more information on these can be found in Chapter \@ref(c05). This decision of which sampling method to use, impacts *sampling error* and can be adjusted for in weighting.
```

> #### Example: Number of Pets in a Household {- #overview-design-sampdesign-ex}

Reviewer: ```suggestion
#### Example: Number of Pets in a Household {.unnumbered #overview-design-sampdesign-ex}
```

To label unnumbered sections, need this syntax instead of using `-` However, you still can't reference this later.

> Let's use a simple example where a researcher is interested in knowing the average number of pets in a household. Our researcher will need to consider what the population of interest is for this. Specifically, are they interested in all of the U.S., a different country, or a more local area (e.g., city or state). Let's assume our researcher is interested in the number of pets in a U.S. household where there is at least one adult living (18 years old or older). In this case, using a sampling frame of mailing addresses would provide the least amount of coverage error. The United States Postal Service (USPS) creates a file of mailing addresses in the U.S. that covers about 95% of U.S. households ({cite Iannichone<!--Check the 95% number when adding citation, not sure if coverage has increased since then-->}). This file is called the Delivery Sequence File (DSF) and is used by many researchers and government agencies to reach the U.S. population. To sample these individuals, for simplicity, we will use a stratified simple random sample design, where we stratify by state. 

Reviewer: Coverage has increased, I will look for a resource.

```suggestion
Let's use a simple example where a researcher is interested in knowing the average number of pets in a household. Our researcher will need to consider what the population of interest is for this. Specifically, are they interested in all of the U.S., a different country, or a more local area (e.g., city or state). Let's assume our researcher is interested in the number of pets in a U.S. household where there is at least one adult living (18 years old or older). In this case, using a sampling frame of mailing addresses would provide the least amount of coverage error. The United States Postal Service (USPS) creates a file of mailing addresses in the U.S. that covers about 95% of U.S. households ({cite Iannichone<!--Check the 95% number when adding citation, not sure if coverage has increased since then-->}). This file is called the Computerized Delivery Sequence File and is used by many researchers and government agencies to reach the U.S. population. To sample these individuals, for simplicity, we will use a stratified simple random sample design, where we stratify by state. 
```

Throughout this chapter we will build on this example research question to plan a survey. 

### Data Collection Planning {#overview-design-dcplanning}

With the sampling design decided, researchers can then make decisions on how to survey these individuals. Specifically, the *mode* used for contacting and surveying the sample, how frequently to send reminders and follow-ups, and the overall timeline of the study are three of the major data collection determinations. Traditionally, researchers have considered four main modes^[Other modes such as using mobile apps or text messaging can also be considered, but have typically smaller reach or are better for longitudinal studies (i.e., surveying the same individuals over many waves of a single study). For the purposes of this overview we will focus on these four main modes.] for conducting surveys:

> -  Computer Assisted Personal Interview (CAPI; also known as face-to-face or in-person interviewing)

Reviewer: Nitpicky but pandemic taught us CAPI can be done via teleconference software, e.g. Zoom

Author: I think it is too new right now for it to be considered at this time.  There might be some research on this at AAPOR this year, so I'm going to hold off until I get a better gauge on how the field is going.  The biggest issue using Zoom is technology barriers that things like CAPI and CATI don't have.  So if we do include it, it would most likely be a 5th mode.

-  Computer Assisted Telephone Interview (CATI; also known as phone or telephone interviewing)

> -  Web

Reviewer: ```suggestion
-  Computer Assisted Web Interviewing (CAWI)
```

> -  Paper and Pencil

Reviewer: ```suggestion
-  Paper and Pencil (PAPI)
```

### Questionnaire Design {#overview-design-questionnaire}

When developing the questionnaire, it can be helpful to first outline the topics to be asked and include the "why" each question or topic is important to the research question(s). This can help researchers better tailor the questionnaire and potentially reduce the number of questions (and thus burden on the respondent) if topics are deemed irrelevant to the research question. When making these decisions, researchers should also consider questions needed for weighting purposes. While we would love to have everyone sampled answer our survey, this is rarely the case. Thus, including questions about demographics in the survey can assist with weighting for *nonresponse error* (both unit and item nonresponse). Knowing details of the sampling plan and what may impact *coverage error* and *sampling error* can help guide researchers in determining what types of demographics to include.

> Researchers can benefit from the work of others by using questions from other surveys. This is common with demographic questions such as race and ethnicity that use questions from a government census or other official surveys. Other survey questions can be found using question banks which are a compilation of questions that have been asked across a variety of surveys such as the [Inter-university Consortium for Political and Social Research (ICPSR) variable search] (https://www.icpsr.umich.edu/web/pages/ICPSR/ssvd/).

Reviewer: I've moved URL to footnote so it works in both printed and web versions. Let's discuss in meeting on how we want to link to external URLs. Footnotes or something else?

```suggestion
Researchers can benefit from the work of others by using questions from other surveys. This is common with demographic questions such as race and ethnicity that use questions from a government census or other official surveys. Other survey questions can be found using question banks which are a compilation of questions that have been asked across a variety of surveys such as the Inter-university Consortium for Political and Social Research (ICPSR) variable search^[See: https://www.icpsr.umich.edu/web/pages/ICPSR/ssvd/].
```

> If a question does not already exist in a question bank, then researchers can craft their own. When creating their own questions, researchers should start with the research question or topic and attempt to write questions that match the concept. The closer the question asked is to the overall concept the better *validity* there is. For example, if the researcher wants to know how people consume tv series and movies, but only ask a question about how many TVs are in the house, then they would be missing other ways that people watch tv series and movies such as on other devices or at places outside of the home.

Reviewer: Capitalize TV?

```suggestion
If a question does not already exist in a question bank, then researchers can craft their own. When creating their own questions, researchers should start with the research question or topic and attempt to write questions that match the concept. The closer the question asked is to the overall concept the better *validity* there is. For example, if the researcher wants to know how people consume TV series and movies, but only ask a question about how many TVs are in the house, then they would be missing other ways that people watch TV series and movies such as on other devices or at places outside of the home.
```

### Data Cleaning and Imputation {#overview-post-cleaning}

Post survey cleaning and imputation is one of the first steps researchers will do to get the survey responses into a dataset for use by analysts. Data cleaning can comprise of cleaning inconsistent data (e.g., with skip pattern errors or multiple questions throughout the survey being consistent with each other), editing numeric entry or open-ended responses for grammar and consistency, or recoding open-ended questions into categories for analysis. Each project should create their own rules for how to handle different cleaning situations, and there are no specific rules that must be followed. Instead, researchers should use their best judgement in ensuring data integrity remains and all decisions should be documented and available to those using the data in analysis. Each decision a researcher makes has an impact on *processing error*, so often researchers often will have multiple people review these rules or recode open-ended data and adjudicate any differences in an attempt to reduce this error.

> Another crucial step in post survey processing is imputation. Often times there is item nonresponse where respondents do not answer specific questions. If the questions are crucial to analysis efforts or to the research question, researchers will implement imputation in an effort to reduce *item nonresponse error*. However, imputation itself can introduce *processing error* as well, so researchers should consider the overall implications of imputing data compared to having item nonresponse. There are multiple ways that imputation can be conducted, we recommend reviewing other resources like {cite Dever}<!--Stephanie is this the right book or is there a better one on imputation?--> for more information.

Reviewer: Not the Dever book. 

- Kim & Shao, Statistical Methods for Handling Incomplete Data
- Will think of more later.

#### Example: Number of Pets in a Household {- #overview-post-cleaning-ex}

> Let's return to the question we created in \@ref(overview-design-questionnaire-ex). The "other specify" invites respondents to specify the type of animal they prefer to have as a pet. If respondents entered answers such as "puppy", "turtle", "rabit", "rabbit", "bunny", "ant farm", "snake", "Mr. Purr", then researchers may wish to categorize these write-in responses to help with analysis. In this example, "puppy" could be assumed to be a reference to a `Dog`, and could be recoded there. The misspelling of "rabit" could be coded along with "rabbit" and "bunny" into a single category of `Bunny or Rabbit`. These are relatively standard decisions that a researcher could make. The remaining write-in responses could be categorized into a few different ways. "Mr. Purr", which may be someone's reference to their own cat, could be recoded to `Cat`, or it could remain as `Other` or some category that is `Unknown`. Depending on the number of responses related to each of others, they could all be combined into a single `Other` category, or maybe categories such as `Reptiles` or `Insects` could be created. Each of these decisions may impact the interpetation of the data, so our researcher should make sure to document the types of responses that fall into each of the new categories.

Reviewer: Don't use reference and instead directly call out the "Number of Pets in a Household". The link will work now but will resolve to ?? since it isn't a numbered section

#### Example: Number of Pets in a Household {- #overview-post-weighting-ex}

In the simple example of our survey, we decided to use a stratified sample by state to select our sample members.  Knowing this sampling design our researcher can include selection weights for analysis that account for how the sample members were selected into the survey.  Additionally, the sampling frame may have the type of building associated with each address, so we could include the building type as a potential nonresponse weighting variable, along with some interviewer observations that may be related to our research topic of average number of pets in a household.  Combining these weights we could create an analytic weight that researchers can use when analyzing the data.

### Disclosure {#overview-post-disclosure}

> Before data is allowed to be made publicly available, researchers will need to ensure that individual respondents can not be identified by the data.  There are a variety of different methods that can be used, but one common method is swapping records.  Researchers may swap specific data values across different respondents such that it does not impact insights that come from the data, but ensures that specific individuals cannot be identified.  Other methods may include aggregating various categories of responses or location information to avoid having only a few people in a given group that could be identified.  There is as much art as there is science to the methods used for disclosure, and in documentation researchers should only provide a high level comments that disclosure was conducted and not specific details to ensure nobody can reverse the disclosure and thus identify individuals.  For more information on different disclosure methods please see {CITE??--Stephanie?}.

Reviewer: Some references

- Skinner chapter in this book: https://www.sciencedirect.com/science/article/abs/pii/S0169716108000151
- AAPOR checklist/standards: https://www-archive.aapor.org/Standards-Ethics/AAPOR-Code-of-Ethics/Survey-Disclosure-Checklist.aspx 
- More to be added

> Before data is allowed to be made publicly available, researchers will need to ensure that individual respondents can not be identified by the data.  There are a variety of different methods that can be used, but one common method is swapping records.  Researchers may swap specific data values across different respondents such that it does not impact insights that come from the data, but ensures that specific individuals cannot be identified.  Other methods may include aggregating various categories of responses or location information to avoid having only a few people in a given group that could be identified.  There is as much art as there is science to the methods used for disclosure, and in documentation researchers should only provide a high level comments that disclosure was conducted and not specific details to ensure nobody can reverse the disclosure and thus identify individuals.  For more information on different disclosure methods please see {CITE??--Stephanie?}.

Reviewer: ```suggestion
Before data is allowed to be made publicly available, researchers will need to ensure that individual respondents can not be identified by the data when confidentiality is required.  There are a variety of different methods that can be used, including data swapping, top or bottom coding, coarsening, and perturbation.  In data swapping, researchers may swap specific data values across different respondents such that it does not impact insights that come from the data, but ensures that specific individuals cannot be identified.  For extreme values, top and bottom coding is sometimes used. For example, very high earners may have their incomes listed as $99,999,999 for any household reporting $99,999,999 or more for their income which is an example of top-coding. Other methods may include aggregating various categories of responses or location information to avoid having only a few people in a given group that could be identified such as including only categorical income instead of continuous which is a form of coarsening.  Data producers may also perturb the data by adding random noise. There is as much art as there is science to the methods used for disclosure, and in documentation researchers should only provide a high level comments that disclosure was conducted and not specific details to ensure nobody can reverse the disclosure and thus identify individuals.  For more information on different disclosure methods please see {CITE??--Stephanie?}.
```
